apps/daimo-mobile/src/view/screen/send/SearchTab.tsx

```diff
@@ -110,8 +110,7 @@ function NoSearchResults() {
   const nav = useNav();
   const sendPaymentLink = () =>
     nav.navigate("SendTab", {
-      screen: "SendNav",
-      params: { autoFocus: true, sendNote: true },
+      screen: "SendLink",
```

Comment:
as long as back button, exit, etc works and the number field is autofocused as soon as you tap "Send Link", looks good to me